maas/data_source_maas_boot_source_selection.go

```diff
@@ -56,6 +56,11 @@ func dataSourceMaasBootSourceSelection() *schema.Resource {
 func dataSourceMaasBootSourceSelectionRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(*ClientConfig).Client
 
+	err := awaitImportComplete(client)
```

Comment:
is there a reason to wait for the import to be completed? the selections are available instantly. In addition, if the Terraform practitioner wants to ensure that the boot resources are available, they have to use the dedicated data_source or resource for them.